com.unity.ml-agents/Documentation~/com.unity.ml-agents.md

```diff
@@ -1,30 +1,30 @@
 # About ML-Agents package (`com.unity.ml-agents`)
 
-The Unity ML-Agents package contains the C# SDK for the
```

Comment:
does this file need to be renamed to `index.md` as well?

Reply:
It only needs to become index.md when you add more markdown files (and a TableOfContents.md file).